spec/plans/report.fmf

```diff
@@ -62,14 +62,28 @@ description:
         - implemented-by: /tmt/steps/report/junit.py
 
 /polarion:
-    summary: Generate a xUnit file and export it into Polarion
+    summary: Generate an xUnit file and export it into Polarion
     story:
         As a tester I want to review tests in Polarion
         and have all results linked to existing test cases there.
     description: |
-        Create a xUnit file ``xunit.xml`` with test results
-        and Polarion properties so the xUnit can then be
-        exported into Polarion.
+        Write test results into a xUnit file and upload to Polarion.
+
+        In order to get quickly started create a pylero config
+        file ``~/.pylero`` in your home directory with the
+        following content:
+
+        .. code-block:: ini
+
+            [webservice]
+            url=https://{your polarion web URL}/polarion
+            svn_repo=https://{your polarion web URL}/repo
+            token={your personal access token}
```

Comment:
This is a bit misleading, since token won't do you any good for report plugin, you need (automation) account with password enabled

Reply:
I see. Thanks for clarification. For `test export` the token is needed, right? Tried to address this in 17d3e1bb. Please, have a look.

Reply:
it's not about the token, but about access, for report you need regular authentication but it doesn't require any permissions, unlike for test export/import where you can authenticate whatever you like (or are allowed to by administrators who can disable password authentication), but you need permissions to the project you are trying to create the test case in

either way, with the commit mentioned it looks good, token is of course preferred